app/models/direct_file_data.rb

```diff
@@ -253,6 +254,14 @@ def fed_taxable_income=(value)
     write_df_xml_value(__method__, value)
   end
 
+  def fed_taxable_pensions
+    df_xml_value(__method__)&.to_i || 0
```

Comment:
oh actually i think `nil&.to_i` is `nil` and `nil.to_i` is 0 lol

Reply:
^ so could simplify to `df_xml_value(__method__).to_i` BUT i think i'm going to leave it as is for now because that exact line is duplicated 34 times across this file 🤦🏻‍♀️ so when (🤞) i get around to a refactor it'll be easier to find and replace